routers/web/repo/projects.go

```diff
@@ -605,19 +604,26 @@ func MoveIssueAcrossBoards(ctx *context.Context) {
 		}
 	}
 
-	issue, err := models.GetIssueByID(ctx.ParamsInt64(":index"))
-	if err != nil {
-		if models.IsErrIssueNotExist(err) {
-			ctx.NotFound("", nil)
-		} else {
-			ctx.ServerError("GetIssueByID", err)
+	form := web.GetForm(ctx).(*forms.MoveProjectIssuesForm)
+
+	issues := make(map[int64]*models.Issue)
+	for _, i := range form.Issues {
+		issue, err := models.GetIssueByID(i.IssueID)
```

Comment:
Not a good idea to select one by one. Select required columns with `WHERE IN` will reduce database workload and reduce latency.